### PR TITLE
Feature/df 1927  kofax logging

### PR DESCRIPTION
**Pull request by emina-str**
add more logging

### CODE REVIEW

File: app-kofax-exporter/src/main/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/messagebroker/DomainEventReceiverGateway.java [hunk 1]
```diff
@@ -26,6 +30,7 @@ public DomainEventReceiverGateway(ExportHandler exportHandler) {
 	@StreamListener(MessageBrokerChannels.APPLICATION_DOCUMENT_EVENTS)
 	@Transactional
 	public void handleApplicationDocumentEvent(@Valid ApplicationDocumentEventDto applicationDocumentEventDto) throws IOException {
+		LOGGER.info("Get event : {} with ID {}", ApplicationDocumentEventDto.class.getSimpleName(), applicationDocumentEventDto.getId());
```
Comment (cjpsycos) [inline app-kofax-exporter/src/main/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/messagebroker/DomainEventReceiverGateway.java]: I would also add an end log. so we can see when the whole handler call finished

File: app-kofax-exporter/src/main/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/messagebroker/DomainEventReceiverGateway.java [hunk 2]
```diff
@@ -39,7 +44,9 @@ public void handleApplicationDocumentEvent(@Valid ApplicationDocumentEventDto ap
 	@StreamListener(MessageBrokerChannels.PROOF_OF_WORK_EFFORTS_EVENTS)
 	@Transactional
 	public void handleProofOfWorkEffortsEvent(@Valid ProofOfWorkEffortsEventDto proofOfWorkEffortsEventDto) throws IOException {
+		LOGGER.info("Get event : {} with ID {}", ProofOfWorkEffortsEventDto.class.getSimpleName(), proofOfWorkEffortsEventDto.getId());
```
Comment (cjpsycos) [inline app-kofax-exporter/src/main/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/messagebroker/DomainEventReceiverGateway.java]: I would also add an end log. so we can see when the whole handler call finished

File: app-kofax-exporter/src/main/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/messagebroker/DomainEventReceiverGateway.java (hunk 1 above)
Comment (emina-str) [reply app-kofax-exporter/src/main/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/messagebroker/DomainEventReceiverGateway.java]: added

File: app-kofax-exporter/src/main/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/messagebroker/DomainEventReceiverGateway.java (hunk 2 above)
Comment (emina-str) [reply app-kofax-exporter/src/main/java/ch/admin/seco/alv/service/onlineform/kofax/exporter/messagebroker/DomainEventReceiverGateway.java]: added

**Pull request merged**